x-pack/plugins/osquery/public/scheduled_query_groups/queries/use_scheduled_query_group_query_form.tsx

```diff
@@ -32,12 +35,36 @@ export interface ScheduledQueryGroupFormData {
 }
 
 export const useScheduledQueryGroupQueryForm = ({
+  scheduledQueryGroupId,
   defaultValue,
   handleSubmit,
-}: UseScheduledQueryGroupQueryFormProps) =>
-  useForm<OsqueryManagerPackagePolicyConfigRecord, ScheduledQueryGroupFormData>({
+}: UseScheduledQueryGroupQueryFormProps) => {
+  const { data } = useScheduledQueryGroups();
+  const ids = useMemo<string[]>(
+    () =>
+      data?.items
+        .find((value) => value.id === scheduledQueryGroupId)
```

Comment:
as we are validating it now against the same `scheduledQueryGroupId` maybe we could just pass `idSet` directly to `useScheduledQueryGroupQueryForm` ?

Reply:
currently `idSet` is only used by the form to validate, the hook retrieving the saved queries doesn't do any of that validation. were you thinking that the hook could be responsible for populating the set and pass it back?

i'm going to remove the `find` call and pass that into the hook at the least, as that will be less wasteful.